lib/r/source.dart

```diff
@@ -103,6 +104,25 @@ Future<void> rSource(
   WidgetRef ref,
   List<String> scripts,
 ) async {
+  // Load the partition train value from shared preferences and update the provider.
+  // So that the values are immediately available on startup.
+
+  final prefs = await SharedPreferences.getInstance();
+  final trainValue = prefs.getInt('train') ?? 60;
```

Comment:
Shouldn't these default values come from the default for the provider? I've updated to fix it. Please confirm it is okay.